diff --git a/build/SQLiteDatabase.js b/build/SQLiteDatabase.js
index 93f11a15c56906b97f993ceb4fdbae320651e2d1..22155fb130e5db22a184dfd9c56f622db05554b5 100644
--- a/build/SQLiteDatabase.js
+++ b/build/SQLiteDatabase.js
@@ -1,6 +1,6 @@
 import { EventEmitter } from 'expo-modules-core';
-import ExpoSQLite from './ExpoSQLiteNext';
-import { SQLiteStatement, } from './SQLiteStatement';
+import ExpoSQLite from './ExpoSQLiteNext.js';
+import { SQLiteStatement, } from './SQLiteStatement.js';
 const emitter = new EventEmitter(ExpoSQLite);
 /**
  * A SQLite database.
diff --git a/build/SQLiteStatement.js b/build/SQLiteStatement.js
index c9e73f535dc3c0d764ac05b34b837357b994560a..7c563dca647621f25e5b894fa36818f592a037a5 100644
--- a/build/SQLiteStatement.js
+++ b/build/SQLiteStatement.js
@@ -1,4 +1,4 @@
-import { composeRow, composeRows, normalizeParams } from './paramUtils';
+import { composeRow, composeRows, normalizeParams } from './paramUtils.js';
 /**
  * A prepared statement returned by [`SQLiteDatabase.prepareAsync()`](#prepareasyncsource) or [`SQLiteDatabase.prepareSync()`](#preparesyncsource) that can be binded with parameters and executed.
  */
diff --git a/build/hooks.js b/build/hooks.js
index 1d19fbb92bd5ecabb0b35948bae320651e2d1..88513baf3c97dadbd1534fcf45c74b3b37bf5f18 100644
--- a/build/hooks.js
+++ b/build/hooks.js
@@ -1,7 +1,7 @@
 import { Asset } from 'expo-asset';
 import React, { createContext, useContext, useEffect, useRef, useState } from 'react';
-import ExpoSQLite from './ExpoSQLiteNext';
-import { openDatabaseAsync } from './SQLiteDatabase';
+import ExpoSQLite from './ExpoSQLiteNext.js';
+import { openDatabaseAsync } from './SQLiteDatabase.js';
 /**
  * Create a context for the SQLite database
  */
diff --git a/build/index.js b/build/index.js
index 0490c653673d646a2780adc2e83b5b63e83b444..49602aa5fa19fe8226e5bcfb4b0498ead5b69bcc 100644
--- a/build/index.js
+++ b/build/index.js
@@ -1,4 +1,4 @@
-export * from './SQLiteDatabase';
-export * from './SQLiteStatement';
-export * from './hooks';
+export * from './SQLiteDatabase.js';
+export * from './SQLiteStatement.js';
+export * from './hooks.js';
 //# sourceMappingURL=index.js.map
